integration-tests/tests/update_os_test.go

```diff
@@ -69,9 +70,9 @@ func (s *updateOSSuite) assertBootDirContents(c *check.C) {
 func (s *updateOSSuite) TestUpdateToSameReleaseAndChannel(c *check.C) {
 	if common.BeforeReboot() {
 		updateOutput := updates.CallFakeOSUpdate(c)
-		expected := "(?ms)" +
-			".*" +
-			"^Reboot to use ubuntu-core version .*\\.\n"
+		expected := fmt.Sprintf("(?ms)"+
+			".*"+
+			"^Reboot to use %s version .*\\.\n", common.OSSnapName(c))
```

Comment:
I would leave the first two lines outside the fmt.Sprintf call, but it's ok as it is